integration-tests/init.spec.js

```diff
@@ -84,30 +83,7 @@ function testRuntimeVersionChecks (arg, filename) {
       }
     }
 
-    if (!currentVersionCanLog) {
```

Comment:
Removed this test since it's effectively replaced by the "unsupported guardrails" tests for now until we rewrite these tests entirely.